### DISCUSSION TITLE
Studio defects

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#14533] [General]
I have noticed two minor issues in the Studio:

1. Sorting indicators have stopped working. Not noticeable in Chrome (disappeared), a minor glitch in Firefox:

![image](https://user-images.githubusercontent.com/7997055/178672978-703364c1-72dc-4b98-b221-a6f6c35b3222.png)

2. Index name suggestion is off by one character (both Firefox and Chrome):

![image](https://user-images.githubusercontent.com/7997055/178673099-62c127b6-c78c-4d9c-9622-c8681ec7f633.png)

**Comment by ppekrol** [comment]
Thanks @krivulcik 
@ml054 Can you take a look?

**Comment by Danielle9897** [comment]
Created:

1. https://issues.hibernatingrhinos.com/issue/RavenDB-18940/Server-Dashboard-Fix-sorting-icons
2. https://issues.hibernatingrhinos.com/issue/RavenDB-18939/Query-view-Fix-index-name-suggestion
